.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Documentation
+about: Improve an existing feature or workflow
+labels: ['kind/documentation', 'status/triage']
+---
+
+### What do you want to improve?
+
+### What is the current documentation?
```

Comment:
What is the goal of this question? Do you want the user to paste the current documentation? The PR will include the current and future changes anyways.